### PR TITLE
Pink Petals Block Mappings

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
All is in the title ^^/.

It's a tiny PR ^^.

### CODE REVIEW

**Comment by Mysterious-Dev** [comment]
That's weird, I don't see why jobs failed...

**Comment by Mysterious-Dev** [comment]
Edit: I see why, shame on me.

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/block/PinkPetalsBlock.mapping [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_onsqqphm net/minecraft/block/PinkPetalsBlock
+	FIELD f_crwyarkl MAX_PETALS_AMOUNT I
```
Comment (eristhea) [inline mappings/net/minecraft/block/PinkPetalsBlock.mapping]: the grammar here is a little strange, I'd prefer to go with singular petal for `MAX_PETAL_AMOUNT`
same goes for min petals amount

Comment (Mysterious-Dev) [reply mappings/net/minecraft/block/PinkPetalsBlock.mapping]: It's changed ^^.

**Review by eristhea — APPROVED** [review]

**Review by EnnuiL — APPROVED** [review]

**Pull request merged**